src/name.cpp

```diff
@@ -1,164 +1,154 @@
 #include <map>
-#include <sstream>
+#include <string>
 
-#include "json.h"
 #include "name.h"
+#include "json.h"
 #include "string_formatter.h"
-#include "output.h"
 #include "translations.h"
 #include "rng.h"
 #include "cata_utility.h"
 
-NameGenerator::NameGenerator()
+static inline nameFlags operator&( nameFlags l, nameFlags r )
```

Comment:
`static` *and* `inline` is a bit pointless. `static` means it only exists in this translation unit, but `inline` means it exists identical in several units. What is it?